src/app/components/Text/index.jsx

```diff
@@ -20,9 +20,27 @@ const StyledParagraph = styled.p`
 const Italic = ({ children }) => <i>{children}</i>;
 const Bold = ({ children }) => <b>{children}</b>;
 
-const Text = ({ text, options }) => {
+const Text = ({ text, paragraphOverride }) => {
```

Comment:
This should have a snapshot test in the `Text/index.test.jsx` file with the `paragraphOverride` prop being used.

Reply:
Done 😎

Reply:
https://github.com/bbc/simorgh/pull/567/files#diff-aa9b8d13ef83d3888b52c8663a01def0R40